observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| statusCode                                           | fixed value = `"COMPLETE"`                                                                                                                                                      |
+| effectiveTime                                        | `Observation.effectiveDateTime` or else `Observation.effectivePeriod`                                                                                                           |
+| availabilityTime                                     | `Observation.effectiveDateTime` or else `Observation.effectivePeriod.start`                                                                                                     | 
+| value                                                | `Observation.valueQuantity` or `Observation.valueString`                                                                                                                        |
+| pertinentInformation / sequenceNumber \[@value]      | fixed value = `"+1"`                                                                                                                                                            |
+| pertinentInformation / pertinentAnnotation / text    | `Observation.bodySite.text` or else `Observation.bodySite.coding.extension[index].value` where `Observation.bodySite.coding.extension[index].url` equals `"descriptionDisplay"` |
+| interpretationCode \[@code]                          | `Observation.interpretation.coding.code`                                                                                                                                        | 
+| interpretationCode \[@codeSystem]                    | fixed value = `"2.16.840.1.113883.2.1.6.5"`                                                                                                                                     |
+| interpretationCode \[@displayName]                   | description of `Observation.interpretation.coding.code` e.g. if code = `"HI"` then displayName = `"Above high reference limit"`                                                 |
+| interpretationCode / originalText                    | `Observation.interpretation.coding.display`                                                                                                                                     |
+| referenceRange / referenceInterpretationRange / text | `Observation.referenceRange.text`                                                                                                                                               |
+| referenceRange / referenceInterpretationRang / value | `Observation.referenceRange.value`                                                                                                                                              |
```

Comment:
Looks like there are `<low>` and `<high>` tags populated, are mentioning these useful?  Having an example below would be an alternative.

Reply:
added example, thanks